src/Tribe/Service_Providers/Onboarding.php

```diff
@@ -43,8 +43,8 @@ public function register() {
 	protected function hooks() {
 		add_action( 'tribe_common_loaded', [ $this, 'register_assets' ] );
 
-		add_action( 'admin_enqueue_scripts', $this->container->make( Onboarding_Main::class, 'localize_tour' ) );
-		add_action( 'admin_enqueue_scripts', $this->container->make( Onboarding_Main::class, 'localize_hints' ) );
+		add_action( 'admin_enqueue_scripts', tribe_callback( $this->container->make( Onboarding_Main::class ), 'localize_tour' ) );
```

Comment:
The signature of `tribe_callback` is `tribe_callback( $slug, $method )`; [see here.](https://github.com/the-events-calendar/tribe-common/blob/b56f4f3f3e1b1f048f175ca47704925fc6dcfdcd/src/Tribe/Container.php#L306).
If you pass an eagerly built object to the method, as you're doing here, it will work, but you're defeating the purpose of the `tribe_callback` method.
Use `tribe_callback( Onboarding_Main::class, 'localize_tour' )`.